sysl2/sysl/syslutil/chroot_fs.go

```diff
@@ -70,3 +79,11 @@ func (fs *ChrootFs) Chmod(name string, mode os.FileMode) error {
 func (fs *ChrootFs) Chtimes(name string, atime time.Time, mtime time.Time) error {
 	return fs.fs.Chtimes(fs.join(name), atime, mtime)
 }
+
+func (fs *ChrootFs) openAllowed(name string) (err error) {
+	modulePath := filepath.Join(fs.root, name)
+	if !strings.HasPrefix(modulePath, fs.root) {
+		err = errors.New("import can not go up the directory")
```

Comment:
"permission denied, file outside root" would be a bettre error

Reply:
changed the error, also changed the way how it checks the import module, should be more foolproof.